### DISCUSSION TITLE
Hello community

### DISCUSSION THREAD

**Discussion by Vuthien312** [community/community#119771] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area
Hello community

### Body

I can't delete my payment method. Does anyone here have the same problem? And another way is like this. Thank you everyone

**Comment by Akash1134** [comment]
Hi @Vuthien312 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
